CHANGELOG.md

```diff
@@ -5,6 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.6.0] - 2021-09-15
```

Comment:
^ I versioned it already to avoid the extra roundtrip of a second PR (and since this repo has low activity, it's unlikely doing so will go wrong.)

In any case, the date might be outdated by the time you review the PR. 
Therefore, here are a few fix candidates, from which one can be applied if needed: 
```suggestion
## [2.6.0] - 2021-09-16
```
```suggestion
## [2.6.0] - 2021-09-17
```
```suggestion
## [2.6.0] - 2021-09-20
```
```suggestion
## [2.6.0] - 2021-09-21
```
```suggestion
## [2.6.0] - 2021-09-22
```

Reply:
No worries. For creating a release you can just do git-flow, in any case.

I can't remember if there are branch protection rules for this repo. (will check) but you're an admin. so should be able to override them (no need to create a pull request to close, though). As discussed before (in PM), a CHANGELOG change is mostly not a change (though it'll get reviewed when in `master`, by me, in any case).